packages/components/filters/filters-list/src/filters-list.tsx

```diff
@@ -1,11 +1,135 @@
+import { type ReactNode, useState } from 'react';
+import { css } from '@emotion/react';
+import { MenuProps, MenuListProps } from 'react-select';
+import DropdownMenu from '@commercetools-uikit/dropdown-menu';
+import FilterMenu from '@commercetools-uikit/filter-menu';
+import SecondaryButton from '@commercetools-uikit/secondary-button';
+import SelectInput from '@commercetools-uikit/select-input';
+
+export type TAppliedFilterValue = {
+  filterKey: string;
+  /** does ReactNode | ReactNode[] make sense for single/multiple labels (shown in FilterMenu Chips) */
+  label: ReactNode[];
```

Comment:
I'm guessing there wouldn't be much value in a singleLabel / multiLabel spilt?

Reply:
I'd go with an array of ReactNodes, as it already is for now, at least this is the simplest form that captures both use cases. If there is a need to change, I am not sure a refactor would be too complex.

Reply:
Requiring the label to always be an array allows the component to handle single/multi values gracefully.  It also means that developers in consuming apps will always pass labels into the component in the same shape, which should lead to less divergent data manipulation to build labels on that side.